### PR TITLE
Add required horizon files

### PR DESCRIPTION
**Pull request by alvarosimon**
This depends on these packages:

* openstack-dashboard
* openstack-manila-ui

### CODE REVIEW

File: openstack-dashboard-vsc.spec [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+install -m 0644 panels/_00_vsc.py $RPM_BUILD_ROOT%{localsettings}/_00_vsc.py
+install -m 0644 panels/__init__.py $RPM_BUILD_ROOT%{vscpanelpath}/__init__.py
+install -m 0644 panels/_disabled.py $RPM_BUILD_ROOT%{vscpanelpath}/_disabled.py
+install -m 0644 img/favicon-vsc.ico $RPM_BUILD_ROOT%{imagespath}/favicon-vsc.ico
+install -m 0644 img/logo-splash-vsc.svg $RPM_BUILD_ROOT%{imagespath}/logo-splash-vsc.svg
+install -m 0644 img/logo-vsc.svg $RPM_BUILD_ROOT%{imagespath}/logo-vsc.svg
+
+
+%post
```
Comment (stdweird) [inline openstack-dashboard-vsc.spec:77]: don't do this in post. i think you can do this in the files section

Comment (stdweird) [reply openstack-dashboard-vsc.spec:77]: see https://stackoverflow.com/questions/32067277/symlink-in-particular-directory-using-rpm-spec/32073947#32073947
create the link in `%install`, and add it to `%files`

**Comment by stdweird** [comment]
@alvarosimon have you tested the rpm somewhere?

File: openstack-dashboard-vsc.spec (hunk 1 above)
Comment (alvarosimon) [reply openstack-dashboard-vsc.spec:77]: ah yes, I will try that

**Comment by alvarosimon** [comment]
@stdweird yes, in `stack01`   I have installed the testing package ` openstack-dashboard-vsc`    and it works like a charm there (take a look to the dashboard)

just one thing is missing, the vsc logos. I can't overwrite directly the logos provided by the `openstack-dashboard`    package, I get an error (I did that manually in cloud.vscentrum as a quick workaround), so I have included the logos in the same directory but with different names (-vsc). 

I took a look to the documentation and it seems that you can choose different logos if you override the css conf. We should create a new css file to override the current logo setup, that is the missing part...

**Comment by stdweird** [comment]
@alvarosimon what documentation?

**Comment by alvarosimon** [comment]
@stdweird here https://docs.openstack.org/horizon/rocky/admin/customize-configure.html

**Comment by stdweird** [comment]
@alvarosimon then we still need to modify/ship the `_stylesheets.html` file. or can we choose that one via a config file?

**Comment by alvarosimon** [comment]
@stdweird we should create a new dahsboard theme (lets say vsc) and set this theme as default from the conf, we can do that. That's is required if we want to change the colors/icons/fonts...
This is an ubuntu example:
https://cloud.garr.it/support/kb/cloud/Customize_dashboard/
By default the OS dashboard provides 2 themes `default` and `material`    we can create a new one `vsc`  and load the logos and change the colors and views if we want.

**Comment by alvarosimon** [comment]
@stdweird finally I found the right documentation for rocky
https://docs.openstack.org/horizon/latest/configuration/themes.html

so we can set a new theme and just include this conf from our `local_settings`   in quattor, and add the css files from the package. I have to test this in stack01

**Comment by alvarosimon** [comment]
ok, we do not need complex things so we can just try this first:
https://docs.openstack.org/horizon/latest/configuration/themes.html#customizing-the-logo

**Comment by alvarosimon** [comment]
@stdweird done, and now the images are correctly updated, we do not need to change the css at all (just for the images I mean). tested in stack01

File: openstack-dashboard-vsc.spec (hunk 1 above)
Comment (alvarosimon) [reply openstack-dashboard-vsc.spec:77]: done

**Pull request merged**